server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+		die();
+	}
+}
+
+function insertBennettMartinLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Bennett Martin data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 7 // Manually obtained from PROD DB
+
+		// Sundays
+		$dates = getWeeklyDatesFromRange('2020-02-02', '2020-03-15')
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 13:00:00", "$date 16:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertFStreetCommunityCenterData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The F Street Community Center data has already been inserted');
+	}
+	
+	try {
+		$DB_CONN->beginTransaction();
+
+        $siteId = 8; // Manually obtained from PROD DB
+
+		// Tuesdays
+        $dates = getWeeklyDatesFromRange('2020-02-04', '2020-04-07');
+        
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 19:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting F Street Community Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertSoutheastCommunityCollegeData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Southeast Community College Data has already been inserted');
+	}
+ 	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = insertSite('Southeast Community College', '8800 O Street, #T-102, Lincoln, NE 68520', '402-472-9638', FALSE, FALSE);
+
+        // Thursdays, closed March 26th, January 30th not on the flyer

```

Comment:
We need to verify the Jan 30 date with Katie as well.

Reply:
Kept the date as the 6th as is. Good call on trusting the flyer over the spreadsheet! Should I delete the part of the comment that mentions the 30?

**"For the Southeast Community College site, the site schedule fliers says that its first Thursday is Feb 6, but the excel sheet said Jan 30. Which is the correct first Thursday date for the Southeast Community College site? FEB 6"**

Reply:
You can delete the part of the comment that mentions Jan 30